src/AppInstallerCLI.sln

```diff
@@ -420,7 +421,8 @@ Global
 		{82B39FDA-E86B-4713-A873-9D56DE00247A}.Release|x86.Build.0 = Release|Win32
 		{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|ARM.ActiveCfg = Fuzzing|x64
 		{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|ARM64.ActiveCfg = Fuzzing|x64
-		{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|x64.ActiveCfg = Fuzzing|x64
+		{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|x64.ActiveCfg = Debug|x64
+		{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|x64.Build.0 = Debug|x64

```

Comment:
>{1622DA16-914F-4F57-A259-D5169003CC8C}.Debug|x64.Build.0 = Debug|x64 [](http://example.com/codeflow?start=2&length=68)

I thought WingetYamlFuzzing proj is only for fuzzing configuration, is it intentional to be included in debug build?

Reply:
Looks like it's intentional after looking at the changes in vcxpeoj

Reply:
Yes, it is intentional and is intended for debugging of specific instances.  Crashes from the fuzzer are easy to debug with the Fuzzing build, but a memory leak is very hard to debug that way since ASAN takes over memory allocation and the tools built into VS won't help as much.